saltstack/base/salt/luks/init.sls

```diff
@@ -0,0 +1,75 @@
+/etc/cdp-luks:
+  file.directory:
+    - name: /etc/cdp-luks
+    - user: root
+    - group: root
+    - mode: 700
+
+/etc/cdp-luks/bin:
+  file.directory:
+    - name: /etc/cdp-luks/bin
+    - user: root
+    - group: root
+    - mode: 700
+
+/etc/cdp-luks/backup:
+  file.directory:
+    - name: /etc/cdp-luks/backup
+    - user: root
+    - group: root
+    - mode: 700
+
+luks_mount_point:
+  file.directory:
+    - name: /mnt/cdp-luks
+    - user: root
+    - group: root
+    - mode: 700
+
+luks_passphrase_tmpfs_mount_point:
+  file.directory:
+    - name: /mnt/cdp-luks_passphrase_tmpfs
+    - user: root
+    - group: root
+    - mode: 700
+
+aws_encryption_sdk_logs:
+  file.directory:
+    - name: /var/log/cdp-luks
+    - user: root
+    - group: root
+    - mode: 700
+
+luks_create_script:
+  file.managed:
+    - name: /etc/cdp-luks/bin/create-luks-volume.sh
+    - source: salt://{{ slspath }}/bin/create-luks-volume.sh
+    - user: root
+    - group: root
+    - mode: 700
+
+luks_reopen_script:
+  file.managed:
+    - name: /etc/cdp-luks/bin/reopen-luks-volume.sh
+    - source: salt://{{ slspath }}/bin/reopen-luks-volume.sh
+    - user: root
+    - group: root
+    - mode: 700
+
+luks_reopen_service:
+  file.managed:
+    - name: /etc/systemd/system/cdp-reopen-luks-volume.service
```

Comment:
Perhaps change the state name to `/etc/systemd/system/cdp-reopen-luks-volume.service`.